src/index.jsx

```diff
@@ -1,17 +1,58 @@
-import React from 'react';
+import React,{ useEffect, useState } from 'react';
 import { render } from 'react-dom';
+
+// styles
 import './style.css';
 
+// components
+import AnimalDetail from './components/AnimalDetail/AnimalDetail';
+import AnimalList from './components/AnimalList/AnimalList';
+
 const App = () => {
+  
+  const [listOfAnimals, setListOfAnimals] = useState([]);
```

Comment:
prázdné pole jako defaultní hodnota je super. Když bys pak třeba chtěla rozlišit mezi stavem "ještě nemám data" a "mám data ale jsou prázdná", máš 2 hlavní možnosti:
- tady dát default state undefined/null
- přidat další useState, který by říkal něco jako [isLoading, setIsLoading] = useState(true);